evaluator/evaluator.py

```diff
@@ -262,6 +291,7 @@ def run(self):
         for sig in signals:
             loop.add_signal_handler(
                 sig, lambda sig=sig: loop.create_task(terminate(sig, loop)))
+        executor = BoundedExecutor(MAX_QUEUE_SIZE, max_workers=WORKER_THREADS)
 
         # pylint: disable=too-many-branches
```

Comment:
Why is pylint disabled here? Isn't it better to write code so you do not need to write disabling annotations here?

Reply:
True, let's address this re-factoring in separate PR.

Reply:
But this is out of the scope of this PR, I guess we could create a new one task to refactor things like this?

Reply:
> True, let's address this re-factoring in separate PR.
Sorry, I don't see this reply :smile: